datumaro/__init__.py

```diff
@@ -1,3 +1,21 @@
-# Copyright (C) 2019-2020 Intel Corporation
+# Copyright (C) 2019-2022 Intel Corporation
 #
 # SPDX-License-Identifier: MIT
+
+import datumaro.components.errors as errors
+import datumaro.components.operations as ops
+import datumaro.components.project as project
+
+from .components.annotation import *
```

Comment:
IMO, we shouldn't use star imports. We might want to have names in the internal modules that are exported to other internal modules, but not to external users.

Reply:
There is a trade-off between having risks of not including something just added and showing internal symbols. In this particular case, I think, there are too many declarations for enumeration.

Reply:
I think explicit imports are more future-proof, though, because if you forget to export something, you can just fix that (by adding it to the list), whereas if you export something you didn't want to, you can't remove it (you can't add exceptions to a star import).

Also, the problem of forgetting to export something can be mitigated by updating tests to import names from `datumaro` whenever possible.

There's also another benefit to explicit imports: they make it easier to see at a glance whether a PR adds or removes public API elements.